unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/HUDController.cs

```diff
@@ -159,16 +157,15 @@ public enum HUDElementID
         CONTROLS_HUD = 18,
         EXPLORE_HUD = 19,
         HELP_AND_SUPPORT_HUD = 20,
-        EMAIL_PROMPT = 21,
-        USERS_AROUND_LIST_HUD = 22,
-        GRAPHIC_CARD_WARNING = 23,
-        BUILDER_IN_WORLD_MAIN = 24,
-        BUILDER_IN_WORLD_INITIAL = 25,
-        QUESTS_PANEL = 26,
-        QUESTS_TRACKER = 27,
-        BUILDER_PROJECTS_PANEL = 28,
-        SIGNUP = 29,
-        COUNT = 30
+        USERS_AROUND_LIST_HUD = 21,
+        GRAPHIC_CARD_WARNING = 22,
+        BUILDER_IN_WORLD_MAIN = 23,
+        BUILDER_IN_WORLD_INITIAL = 24,
+        QUESTS_PANEL = 25,
+        QUESTS_TRACKER = 26,
+        BUILDER_PROJECTS_PANEL = 27,
+        SIGNUP = 28,
+        COUNT = 29
```

Comment:
Please do not break compatibility at any moment with kernel, enums in protocols are constant and forever. You could leave the old unused enum element, it could be documented to avoid usage. 

Removing and reassigning values would create an unnecessary friction in the merge of both pull request simultaneously. And that is something we can avoid.